tasks/sep/zora-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,359 @@
+  **After:** `0x0000000000000000000000c2be75506d5724086deb7245bd260cc9753911be00` <br/>
+  **Meaning:** Sets `superchainConfig` at slot `0x35` (53). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/OptimismPortal.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal.json#L58-L64). The `superchainConfig` address can be found in the [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/configs/sepolia/superchain.yaml#L8).
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000036` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x0000000000000000000000002615b481bd3e5a1c0c7ca3da1bdc663e8615ade9` <br/>
+  **Meaning:** Sets `l2Oracle` at slot `0x36` (54). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/OptimismPortal.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal.json#L66-L70). The `L2OutputOracleProxy` address can be found in the [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/4156733e723282b632a5743ac3064710caea03d4/superchain/extra/addresses/sepolia/zora.json#L6).
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000037` <br/>
+  **Before:** `0x0000000000000000000000000000000000000000000000000000000000000000` <br/>
+  **After:** `0x000000000000000000000000b54c7bfc223058773cf9b739cc5bd4095184fb08` <br/>
+  **Meaning:** Sets `systemConfig` at slot `0x37` (55). The correctness of
+   this slot is attested to in the Optimism repo at [storageLayout/OptimismPortal.json](https://github.com/ethereum-optimism/optimism/blob/op-contracts/v1.3.0/packages/contracts-bedrock/snapshots/storageLayout/OptimismPortal.json#L73-L77). The `SystemConfigProxy` address can be found in the [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/4156733e723282b632a5743ac3064710caea03d4/superchain/extra/addresses/sepolia/zora.json#L10).
+
+- **Key:** `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc` <br/>
+  **Before:** `0x0000000000000000000000008e24bdf3dcba149bae8dd8b73efd9d886f9f5d3c` <br/>
+  **After:** `0x0000000000000000000000002d778797049fe9259d947d1ed8e5442226dfb589` <br/>
+  **Meaning:** Implementation address is set to the new `OptimismPortal`. The implementation address can be found in the [Superchain Registry](https://github.com/ethereum-optimism/superchain-registry/blob/5ad42cbb49472a0bf164ade976426f7526ee6dfe/superchain/implementations/networks/sepolia.yaml#L12).
+
+### `0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2` (The 1 of 1 `ProxyAdmin` owner Safe)
```

Comment:
Looks like this is the proxyAdmin address, not the Safe address (proxy admin owner). It should be 0xE17071F4C216Eb189437fbDBCc16Bb79c4efD9c2, and let's move this above the OptimismPortalProxy to match the Tenderly ordering